_pages/dev/components/banner-component.md

```diff
@@ -2,62 +2,62 @@
 title: Banner Component
 ---
 
-The purpose of the Banner component is to render a media in a given page slot. The media can be optimized for the given slot size. The banner is typically used to link to another internal or external page.
+The banner component is used to render a media item in a given page slot. The media item can be optimized for the given slot size. The banner is typically used to link to another internal or external page.
```

Comment:
The banner component is used to render different banners that are created in the CMS. Banners contain one or multiple images as well as some optional content, such as a header. A banner is often used to link to another content.

Reply:
👍